### PR TITLE
First-frame: Use arguments that throw errors as shown in help menu

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: examples/first-frame/main.cpp [hunk 1]
```diff
@@ -112,42 +112,46 @@ int main(int argc, char *argv[]) {
...
-                  << Help_Menu;
+        LOG(ERROR) << "Argument " << arg_error << " doesn't exist! "
+                   << "Please check help menu.";
         return -1;
     }
+
     result = command.helpMenu();
     if (result == 1) {
-        std::cout << Help_Menu;
+        LOG(INFO) << Help_Menu;
```
Comment (dNechita) [inline examples/first-frame/main.cpp:122]: The help menu should not be printed with the LOG tool. Every command line tool I have used so far has printed the help menu directly without any additional text before or after the menu.

File: examples/first-frame/main.cpp [hunk 2]
```diff
@@ -112,42 +112,46 @@ int main(int argc, char *argv[]) {
     command.parseArguments(argc, argv, command_map);
     int result = command.checkArgumentExist(command_map, arg_error);
     if (result != 0) {
-        std::cout << "Argument " << arg_error << " doesn't exist."
-                  << "\n"
-                  << Help_Menu;
+        LOG(ERROR) << "Argument " << arg_error << " doesn't exist! "
```
Comment (dNechita) [inline examples/first-frame/main.cpp:115]: It's a general practice to show the help menu whenever the user doesn't provide the right arguments to the command line.

**Review by dNechita — CHANGES_REQUESTED** [review]

File: examples/first-frame/main.cpp (hunk 2 above)
Comment (liealex) [reply examples/first-frame/main.cpp:115]: Changed all help menu printing to be done with cout

File: examples/first-frame/main.cpp (hunk 1 above)
Comment (liealex) [reply examples/first-frame/main.cpp:123]: Solved. Now help menu is printed using std::cout

Comment (dNechita) [reply examples/first-frame/main.cpp:123]: Well, it is still being printed with LOG.

Comment (liealex) [reply examples/first-frame/main.cpp]: Fixed

**Review by dNechita — APPROVED** [review]

**Pull request merged**